### DISCUSSION TITLE
Number Input - Design and Spec RFC

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#3280] [Review request]
Hi all, 

Paste is adding a Number Input component and we would like your feedback.

<img width="249" alt="Screenshot 2023-06-12 at 3 07 48 PM" src="https://github.com/twilio-labs/paste/assets/87141551/3d78d218-4cec-4789-8bc0-ed7889dd6009">

### Component spec
Review [the component spec](https://docs.google.com/document/d/17LuSxzX2leV9aozslLIAJgRlK246MAhfdOrRxQgbfIk/edit#) any leave any comments/suggestions/questions directly in the google doc.

### Design
Review the below design proposal and leave any comments in this thread or in the [Figma file](https://www.figma.com/file/SmsNwQqp5rBtwXrmmwzuZO/Number-Input-and-Slider?type=design&node-id=207-54423).

<img width="691" alt="Screenshot 2023-06-12 at 3 08 21 PM" src="https://github.com/twilio-labs/paste/assets/87141551/007735b8-8a61-4543-92b3-e05095001b9b">

Thanks!

**Comment by TheSisb** [comment]
The component has been released and the window for comment has been closed.
